### ISSUE TITLE
Add Normalizer preprocessor

### ISSUE THREAD

**Issue by rhiever** [rhiever/tpot#213]
Add the sklearn [Normalizer](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.Normalizer.html) preprocessor to the primitive set.

The only evolvable parameter would be `norm`: ‘l1’, ‘l2’, or ‘max’

**Comment by rhiever** [comment]
Added in 0.5 release.
